src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,51 @@
+package duke.task;
+
+/**
+ * Represents an event task that has a description, a start date/time, and an end date/time.
+ */
+public class Event extends Task {
+    protected String from;
```

Comment:
Could we use a noun to identify the starting/ending date instead of `from` and `by`?